src/batched_inference_executor.py

```diff
@@ -31,7 +31,7 @@
 import tb_controller_pb2_grpc
 
 
-WARMUP_REQS = 5
+WARMUP_REQS = 2
```

Comment:
Do we actually need warm up requests at all?

Reply:
yes! The first few are extremely slow, that it would benefit if we didn't migrate at all.